eva-submission-docker/eva_docker/Dockerfile

```diff
@@ -0,0 +1,77 @@
+FROM ubuntu:22.04
+
+ENV JAVA_VERSION 8
+ENV NXF_VER 21.10.5
+ENV vcf_validator_version 0.9.4
+
+COPY maven-settings.xml /root
+COPY eva_docker/submission_config.yml /root/.submission_config.yml
+
+RUN mkdir -p /usr/local/test_eva_submission/data
+RUN mkdir -p /usr/local/test_eva_submission/submissions
+RUN mkdir -p /usr/local/eva/public_ftp/
+RUN mkdir -p /usr/local/eva/datasources/reference_sequences
+RUN mkdir -p /usr/local/eva/datasources/vep-cache
+RUN mkdir -p /usr/local/software/opencga
+
+ADD eva_docker/ftp /usr/local/eva/ftp/
+ADD eva_docker/conf /usr/local/software/opencga/conf
+
+# install utility libs
+RUN apt update  \
+    && apt install -y curl wget maven git vim samtools bcftools tabix bedtools minimap2 rsync  \
+    && apt install -y python3-pip python3.10-venv
+
+# install java
+RUN apt install -y openjdk-${JAVA_VERSION}-jdk
+# Set Java Home
+ENV JAVA_HOME /usr/lib/jvm/java-${JAVA_VERSION}-openjdk-amd64
+ENV PATH $JAVA_HOME/bin:$PATH
+
+# install nextflow
+RUN curl -L "https://github.com/nextflow-io/nextflow/releases/download/v${NXF_VER}/nextflow-${NXF_VER}-all" | bash \
+    && mv nextflow /usr/local/bin/
+
+# install mongodb tools
+RUN wget -qO - https://www.mongodb.org/static/pgp/server-4.4.asc | apt-key add -
```

Comment:
Any reason for 4.4? We should try as much as possible to stick to 4.0.28 unless there is a compelling reason not to....

Reply:
Also we should extract MongoDB version as an environment variable similar to Java and Nextflow...

Reply:
1. 4.0 is not compatible with ubuntu 22.04
2. extracted mongo db version as an environment variable